src/views/ipv/page/pyi-triage-select-smartphone.njk

```diff
@@ -0,0 +1,87 @@
+  <ul class="govuk-list govuk-list--bullet ">
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.iphone.condition1' | translate }}</li>
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.iphone.condition2' | translate }}</li>
+  </ul>
+  {{ govukDetails({
+      summaryText: 'pages.pyiTriageSelectSmartphone.content.iphone.infoLabel' | translate,
+      text: 'pages.pyiTriageSelectSmartphone.content.iphone.infoHtml' | translate( { url: contactUsUrl } ) | safe
+  }) }}
+
+  <h2 class="govuk-heading-m">{{ 'pages.pyiTriageSelectSmartphone.content.android.subHeading' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.android.paragraph' | translate }}</p>
+  <ul class="govuk-list govuk-list--bullet ">
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.android.condition1' | translate }}</li>
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.android.condition2' | translate }}</li>
+  </ul>
+  {{ govukDetails({
+    summaryText: 'pages.pyiTriageSelectSmartphone.content.android.infoLabel' | translate,
+    text: 'pages.pyiTriageSelectSmartphone.content.android.infoHtml' | translate( { url: contactUsUrl } ) | safe
+  }) }}
+
+  <h2 class="govuk-heading-m">{{ 'pages.pyiTriageSelectSmartphone.content.subHeadingAppendix' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.paragraphAppendix1' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.paragraphAppendix2' | translate }}</p>
+
+  <form id="pyiTriageSelectSmartphoneOptionsForm" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+    {% set radiosConfig = {
+      idPrefix: "journey",
+      name: "journey",
+      fieldset: {
+        legend: {
+          text: 'pages.pyiTriageSelectSmartphone.content.subHeadingSelectSmartphone' | translate,
+          classes: "govuk-fieldset__legend--m"
+        }
+      },
+      items: [
+        {
+          value: "iphone",
+          text: 'pages.pyiTriageSelectSmartphone.content.formRadioButtons.iphoneButtonText' | translate
+        },
+        {
+          value: "android",
+          text: 'pages.pyiTriageSelectSmartphone.content.formRadioButtons.androidButtonText' | translate
+        },
+        {
+          value: "end",
+          text: 'pages.pyiTriageSelectSmartphone.content.formRadioButtons.neitherButtonText' | translate
+        }
+      ]
+    } %}
+
+    {% if errorState %}
+      {% set errorMessageObject = { 'text': 'pages.pyiTriageSelectSmartphone.content.formErrorMessage.errorRadioMessage' | translate } %}
+      {% set radiosConfig = radiosConfig | setAttribute('errorMessage', errorMessageObject) %}
+    {% endif %}
+
+    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+    {{ govukRadios(radiosConfig) }}
+    {{ govukButton({
+      id: submitButton,
+      classes: "govuk-!-margin-top-7",
```

Comment:
I suspect that the spacing above the button in the desktop UCD screenshot is not deliberate - there is no gap above the button in the mobile version of the page (and the gap looks weird to me anyway).

Reply:
Yeah agreed this class should be removed

Reply:
> I suspect that the spacing above the button in the desktop UCD screenshot is not deliberate - there is no gap above the button in the mobile version of the page (and the gap looks weird to me anyway).

Glad you think so - will remove